lib/facade/MonitoringFacade.ts

```diff
@@ -110,7 +110,7 @@ import {
 import { MonitoringAspect, MonitoringAspectProps } from "./MonitoringAspect";
 
 export interface MonitoringFacadeProps {
-  readonly metricFactoryDefaults: MetricFactoryDefaults;
+  readonly metricFactoryDefaults?: MetricFactoryDefaults;
   readonly alarmFactoryDefaults: AlarmFactoryDefaults;
```

Comment:
Would love to add some defaults for `alarmFactoryDefaults`  to make it further easy to create a Facade

Reply:
Done in the next rev.